docs/src/capi/howto/external-etcd.md

```diff
@@ -0,0 +1,127 @@
+# CAPI External Etcd
+
+To replace the built-in [dqlite][dqlite] database with an external etcd to
+manage the kubernetes state in the CAPI workload cluster follow this
+`how-to guide`.
+This example shows how to create a 3-node workload cluster with an external
+etcd. 
+
+## Prerequisites
+
+To follow this guide, you will need:
+
+- Clusterctl
+- A CAPI management cluster initialized with the infrastructure, bootstrap and
+  control plane providers of your choice
+- Secured 3-node etcd deployment
+
+Please refer to the [getting-started][getting-started] for instructions.
+
+## Create Kubernetes Secrets
+
+Create three kubernetes secrets:
+
```

Comment:
If you want to give an example cluster name, I would start by saying that, then use it for all the examples. It's best if we can avoid <insert-your-data>-type things where possible. If you say e.g. "For this example we will bw given our cluster the name 'agamemnon'" and then using that name. It gets you out of a lot of explaining and funny characters.

Reply:
Say less!